### PR TITLE
[onert][gpu_cl] Add Pool2D operation.

### PR DESCRIPTION
**Pull request by Hyunjun85**
1. Add Pool2d operation.
2. Add testcases.

Signed-off-by: H. Kim hyunjun2.kim@samsung.com
Signed-off-by: sungho22.lee sungho22.lee@samsung.com

### CODE REVIEW

**Review by intom — APPROVED** [review]
It looks good

**Review by juitem — DISMISSED** [review]
LGTM

**Comment by Hyunjun85** [comment]
@nnfw-bot test cross-aarch64-runtime-test-release

**Comment by Hyunjun85** [comment]
@nnfw-bot test cross-runtime-test-debug

**Review by juitem — APPROVED** [review]

**Review by intom — APPROVED** [review]

**Review by YiHyunJin — DISMISSED** [review]

File: runtime/onert/backend/gpu_cl/KernelGenerator.cc [hunk 1]
```diff
@@ -456,6 +469,68 @@ void KernelGenerator::visit(const ir::operation::ElementwiseActivation &node)
...
+  gpu_op->SetSrc(input_tensor->handle(), ir::operation::Pool2D::Input::INPUT);
+  gpu_op->SetDst(output_tensor->handle(), 0);
+
+  fn->configure(_creation_context);
+  fn->add_operation(std::move(gpu_op));
+
+  _return_fn = std::move(fn);
+}
+
+// void visit(const ir::operation::Reshape &)
```
Comment (chunseoklee) [inline runtime/onert/backend/gpu_cl/KernelGenerator.cc:522]: Please remove comments.

File: runtime/onert/backend/gpu_cl/KernelGenerator.h [hunk 2]
```diff
@@ -49,6 +49,10 @@ class KernelGenerator : public basic::KernelGeneratorBase
   void visit(const ir::operation::Conv2D &) override;
   void visit(const ir::operation::DepthwiseConv2D &) override;
   void visit(const ir::operation::ElementwiseActivation &) override;
+  void visit(const ir::operation::Pool2D &) override;
+  // NYI
```
Comment (chunseoklee) [inline runtime/onert/backend/gpu_cl/KernelGenerator.h:53]: Please remove comments

Comment (Hyunjun85) [reply runtime/onert/backend/gpu_cl/KernelGenerator.h:53]: Modified.

File: runtime/onert/backend/gpu_cl/KernelGenerator.cc (hunk 1 above)
Comment (Hyunjun85) [reply runtime/onert/backend/gpu_cl/KernelGenerator.cc:522]: Modified.

**Review by YiHyunJin — APPROVED** [review]

**Review by juitem — APPROVED** [review]

**Review by chunseoklee — APPROVED** [review]

**Review by intom — APPROVED** [review]

**Pull request merged**